net/netutil/ip_forward.go

```diff
@@ -146,6 +147,87 @@ func CheckIPForwarding(routes []netip.Prefix, state *interfaces.State) (warn, er
 	return nil, nil
 }
 
+// CheckReversePathFiltering reports whether reverse path filtering is either
+// disabled or set to 'loose' mode for exit node functionality on any
+// interface.
+//
+// The state param can be nil, in which case interfaces.GetState is used.
+//
+// The routes should only be advertised routes, and should not contain the
+// nodes Tailscale IPs.
+//
+// This function returns an error if it is unable to determine if reverse path
+// filtering is enabled, or a warning describing configuration issues if
+// reverse path fitering is non-functional or partly functional.
+func CheckReversePathFiltering(routes []netip.Prefix, state *interfaces.State) (warn, err error) {
+	if runtime.GOOS != "linux" {
```

Comment:
will this ever be on anything other than Linux? Should the CLI only conditionally call this too to avoid some latency on other OSes?